packages/headless/src/controllers/core/facets/category-facet/headless-core-category-facet.ts

```diff
@@ -141,10 +141,33 @@ export interface CoreCategoryFacetState {
   /** The facet ID. */
   facetId: string;
 
-  /** The facet's parent values. */
+  /**
+   * The facet's values, represented in their hierarchical form
+   * if there's an hierarchy, flat otherwise.
```

Comment:
```suggestion
   * if there's a hierarchy, flat otherwise.
```

Reply:
I find it a bit unclear what "if there's a hierarchy" means here, and what "flat otherwise" means too.

I think we should give some examples of what this value may contain.

Reply:
- [ ] add JSDOC example of possible values, flat and hierarchical

Reply:
Comment has been rewritten/simplified